names/names.py

```diff
@@ -10,11 +10,28 @@
 names_2 = f.read().split("\n")  # List containing 10000 names
 f.close()
 
-duplicates = []
-for name_1 in names_1:
-    for name_2 in names_2:
-        if name_1 == name_2:
-            duplicates.append(name_1)
+
+# Default initial code for duplicates
+# duplicates = []
+# for name_1 in names_1:
+#    for name_2 in names_2:
+#        if name_1 == name_2:
+#            duplicates.append(name_1)
+# runtime - 7.0s
+
+
+# Check out difference removing one of the for loops
+# duplicates = []
+# for name_1 in names_1:
+#    if name_1 in names_2:
+#        duplicates.append(name_1)
+# runtime - 1.3s
+
+
+# Tried using dictionary with union of set operations
+duplicates = set(names_1) & set(names_2)
```

Comment:
This is the cleanest solution I have seen.  Great Job!

Not only does it make use of sets for constant time lookup as well as the intersection operator, but just the way we have `set & set` here in the code seems like a line directly out of a math book which is directly representative of what the problem is asking.  It definitely supports the idea that 'python is executable pseudocode' and reads much better than other implementations of the same operations.